packages/edit-site/src/components/page-pages/index.js

```diff
@@ -231,7 +231,7 @@ export default function PagePages() {
 						</a>
 					);
 				},
-				filters: [ 'enumeration' ],
+				filters: [ 'in' ],
```

Comment:
This is the only change that I was not sure about because maybe enumeration is the "type" we should define and the available operators for that particular filter type should be a nested config or something but this is also a good approach for now, so let's just keep it that way.

Reply:
I gave this a shot at https://github.com/WordPress/gutenberg/pull/55996